### ISSUE TITLE
AsyncStorage(React Native Community version) doesnt support getItem/setItem/removeItem in latest version

### ISSUE THREAD

**Issue by vilvaathibanpb** [apollographql/apollo-cache-persist#325]
`@react-native-community/async-storage@next` doesn't support `setItem`, `removeItem` and `getItem` functions anymore and sticks to `get`, `set` and `remove` APIs. This will break the implementation of apollo-cache-persist and make it completely unusable in latest versions.

**Comment by eduplus** [comment]
This would be good to get fixed! Workaround is to use 1.7.1. instead of 2.0.0 for now.

**Comment by vilvaathibanpb** [comment]
@eduplus I can raise a PR for the same. Do you have a pre-set approach for this or shall I figure out a way to solve this?

**Comment by ChronSyn** [comment]
I don't suppose anyone's got a fork available which fixes this issue (or has some idea on how this can be fixed)?

This library is the only option that I can see which supports persistence in RN, but it is incompatible with current versions of async-storage.

**Comment by wtrocki** [comment]
Workaround. 

```js

asyncStorage.setItem=asyncStorage.set
asyncStorage.getItem=asyncStorage.get
asyncStorage.removeItem=asyncStorage.remove

...
store: asyncStorage
...

```

The way I would see this done in the library is to provide docs/wrapper

**Comment by wtrocki** [comment]
hi lads. We talked about this issue couple times. I think we need one storage interface in core and wrappers for each storage we support. Adding as simple fix

**Comment by wodCZ** [comment]
Seems the async-storage settled with `setItem` API after moving to https://github.com/react-native-async-storage/async-storage from @react-native-community.
At least from what I see at `master` branch.

**Comment by wtrocki** [comment]
Will be fixed in #391
